lib/src/ui/codelessly_widget.dart

```diff
@@ -265,6 +265,14 @@ class _CodelesslyWidgetState extends State<CodelesslyWidget> {
   StreamSubscription<CodelesslyException?>? _exceptionSubscription;
   CodelesslyException? _lastException;
 
+  // Saved in the state for the didChangeDependencies method to use to compare
+  // with the new canvas ID to determine if the layout needs to be reloaded
+  // when the media query changes resulting into a new breakpoint.
+  String? canvasID;
+
+  // Saved in the state for the didChangeDependencies method to use.
+  String? effectiveLayoutID;
```

Comment:
the layoutID never changes. The canvasID changes because a layout may hold different canvases.

What is the point of this?

Reply:
This is because of this!
<img width="562" alt="image" src="https://github.com/Codelessly/CodelesslySDK/assets/20423471/f60db0e2-8158-4ed7-b669-882eebd1121f">

Reply:
This information is only available once the publish model is loaded and available.